src/app/inquiry/page.tsx

```diff
@@ -102,36 +53,26 @@ export default function Inquiry() {
 
   const handleInputChange = (fieldName: string, value: string) => {
     console.log(`${fieldName}: ${value}`);
+    if (fieldName === 'content') {
+      if (!isContentLengthValid(value)) {
+        alert('내용은 2000자를 초과할 수 없습니다.');
+        return;
+      }
+    }
```

Comment:
```suggestion
if(fieldName === 'content' && !isContentLengthValid(value)){
  alert('내용은 2000자를 초과할 수 없습니다.');
  return;
}
```
if문의 조건 2개를 하나로 합칠 수 있습니다.

if문의 조건이 false일때 다른 조건을 확인하는것 이외에 따로 진행되는 로직이 없기때문에 fieldName과 내용의 길이를 한 번에 조건으로 줄 수 있겠네요

근데 isContentLengthValid(value)가 true면 어떻게 되는건가요?

Reply:
isContentLengthValid(value)가 true면 사용자의 입력이 글자 수 제한을 만족하는것으로 되어서 경고 없이 해당 입력값을 입력할수있게 했읍니다